Rest/Controller/ContentController.php

```diff
@@ -131,6 +136,36 @@ public function getContent($contentIdList, Request $request)
         return new ContentDataValue($content);
     }
 
+    /**
+     * Preparing array of integers based on comma separated integers in string or single integer in string.
+     *
+     * @param string $string list of integers separated by comma character
+     *
+     * @return array
+     *
+     * @throws LogicException When incorrect $list value is given
+     */
+    protected function getIdListFromString($string)
+    {
+        if (is_numeric($string)) {
```

Comment:
If we would like to check is` $item` is an integer we can't use `is_numeric` because it also accepts floats. `filter_var($item, FILTER_VALIDATE_INT)` seems to be a better choice.